PersonBook/PersonBook/PersonBook/Service/PersonService.cs

```diff
@@ -31,17 +40,17 @@ public void DeletePerson(int id)
             }
         }
 
-        public List<Person> ListPersons()
+        public ICollection<Person> ListPersons()
         {
 
             return _personList;
         }
 
-        public void UpdatePerson(Person person)
+        public void UpdatePerson(int Id,string name,string surname)
         {
-            Person p = _personList.FirstOrDefault(i => i.Id == person.Id);
-            p.FirstName = person.FirstName;
-            p.LastName = person.LastName;
+            Person p = _personList.FirstOrDefault(i => i.Id == Id);
```

Comment:
that should be `var person = _personList.Fir....`
And you should check whether the person is null or not!

Reply:
Done